src/Behavioral.Automation/Services/Mapping/ScopeContextManager.cs

```diff
@@ -41,5 +43,14 @@ public void SwitchPage(string pageName)
             var pageScopeContext = _uriToPageScopeMapper.GetPageScopeContext(pageName);
             _scopeContextRuntime.SwitchToPageScope(pageScopeContext);
         }
+
+        public void SwitchToCurrentUrl(string currentUrl)
```

Comment:
Can we use `public void SwitchPage(Uri uri)` instead?

Reply:
Done

Reply:
I mean the same method with `Uri` argument.